dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/FrameworkViewPage.ts

```diff
@@ -0,0 +1,562 @@
+import { describeIf } from "@e2e/support/TestUtility";
+import { getBaseUrl, uploader_name, uploader_pw } from "@e2e/utils/Cypress";
+import { getKeycloakToken } from "@e2e/utils/Auth";
+import { FixtureData } from "@e2e/fixtures/FixtureUtils";
+import { DataTypeEnum, EuTaxonomyDataForFinancials, LksgData } from "@clients/backend";
+import { uploadOneEuTaxonomyFinancialsDatasetViaApi } from "@e2e/utils/EuTaxonomyFinancialsUpload";
+import { uploadOneLksgDatasetViaApi } from "@e2e/utils/LksgUpload";
+import { getPreparedFixture } from "@e2e/utils/GeneralApiUtils";
+import { generateDummyCompanyInformation, uploadCompanyViaApi } from "@e2e/utils/CompanyUpload";
+import { uploadOneEuTaxonomyNonFinancialsDatasetViaApi } from "@e2e/utils/EuTaxonomyNonFinancialsUpload";
+import { generateEuTaxonomyDataForNonFinancials } from "@e2e/fixtures/eutaxonomy/non-financials/EuTaxonomyDataForNonFinancialsFixtures";
+import { humanizeString } from "../../../../src/utils/StringHumanizer";
+import { uploadOneSfdrDataset } from "../../utils/SfdrUpload";
+import { generateSfdrData } from "../../fixtures/sfdr/SfdrDataFixtures";
+import { generateLksgData } from "../../fixtures/lksg/LksgDataFixtures"; // TODO write paths with @notation!
+
+describe("The shared header of the framework pages should act as expected", { scrollBehavior: false }, () => {
```

Comment:
This test seems rather excessive. Lets discuss it.

Reply:
Feel free to call me for this one.  This is a huge topic.